.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,26 @@
-^ Make sure to prefix the Title with [WIP]/[WIPTEST]
+<!-- Make sure to prefix the PR Title in the following flow...
```

Comment:
Suggestion : 

           
                 <!-- Make sure to prefix the PR Title with any one of the below options..